activestorage/test/dummy/config/application.rb

```diff
@@ -19,8 +19,10 @@
 
 module Dummy
   class Application < Rails::Application
-    config.load_defaults 5.1
+    config.load_defaults 5.2
 
     config.active_storage.service = :local
+
+    config.action_controller.default_protect_from_forgery = false
```

Comment:
Looks like activestorage doesn't work when `config.action_controller.default_protect_from_forgery = true`. It is `true` by default for 5.2